srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/UtilizationSummary.java

```diff
@@ -0,0 +1,157 @@
+package gov.va.med.srcalc.web.view.admin;
+
+import java.util.Collection;
+import java.util.Objects;
+
+import com.google.common.base.MoreObjects;
+
+import gov.va.med.srcalc.domain.calculation.HistoricalCalculation;
+import gov.va.med.srcalc.domain.calculation.HistoricalRunInfo;
+import gov.va.med.srcalc.domain.calculation.SignedResult;
+
+/**
+ * <p>Encapsulates summary statistics for a Utilization Report.</p>
+ * 
+ * <p>Per Effective Java Item 17, this class is marked final because it was not
+ * designed for inheritance.</p>
+ */
+public final class UtilizationSummary
+{
+    private final int fTotalCount;
+    private final int fSignedCount;
+    private final int fSecondsToFirstRunAverage;
+    private final int fSecondsToSignAverage;
+    
+    /**
+     * Constructs an instance with the given properties.
+     * @param totalCount
+     * @param signedCount
+     * @param secondsToFirstRunAverage
+     * @param secondsToSignAverage
+     */
+    public UtilizationSummary(
+            final int totalCount,
+            final int signedCount,
+            final int secondsToFirstRunAverage,
+            final int secondsToSignAverage)
+    {
+        fTotalCount = totalCount;
+        fSignedCount = signedCount;
+        fSecondsToFirstRunAverage = secondsToFirstRunAverage;
+        fSecondsToSignAverage = secondsToSignAverage;
+    }
+    
+    /**
+     * Constructs an instance, calculating statistics from the given HistoricalRunInfo
+     * objects.
+     * @return a new instance encapsulating the statistics
+     */
+    public static UtilizationSummary fromRunInfos(
```

Comment:
Nah, we haven't been enforcing that, there are probably hundreds of missing `@param` docs.